.github/workflows/releases-json.yml

```diff
@@ -0,0 +1,55 @@
+name: releases-json
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  release:
+    types:
+      - released
+  pull_request:
+    paths-ignore:
+      - '.github/releases.json'
+
+jobs:
+  generate:
+    uses: crazy-max/.github/.github/workflows/releases-json.yml@002654044825b3c2b9856af61b8a2aaf389706b1
```

Comment:
Nice, good to have something reusable :sweat_smile: 

Tiny nit - could we potentially have this produce indented JSON, it would be useful if it was human-readable.

Reply:
> Tiny nit - could we potentially have this produce indented JSON, it would be useful if it was human-readable.

Yes might be better, done.